multirun/process.go

```diff
@@ -81,6 +81,10 @@ func (p *process) run(ctx context.Context) error {
 	return nil
 }
 
+func isFinished(err error) bool {
+	return err.Error() == "os: process already finished"
```

Comment:
correct me if I'm wrong but it doesn't seem to be any better way because the instance of the error that got returned is package private unfortunately :(